lib/hyacinth/adapters/external_identifier_adapter/datacite.rb

```diff
@@ -4,109 +4,136 @@ module Hyacinth
   module Adapters
     module ExternalIdentifierAdapter
       class Datacite < Abstract
-        IDENTIFIER_STATUS = { public: 'public',
-                              reserved: 'reserved',
-                              unavailable: 'unavailable' }.freeze
+        DOI_STATES = [:draft, :findable, :registered].freeze
+
+        # https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf
+        REQUIRED_DATACITE_METADATA_PROPERTIES = [:creators,
+                                                 :publisher,
+                                                 :publication_year,
+                                                 :resource_type_general,
+                                                 :title,
+                                                 :url].freeze
+
+        # https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf
+        SUPPORTED_DATACITE_METADATA_PROPERTIES = [:subject,
```

Comment:
Is this `SUPPORTED_DATACITE_METADATA_PROPERTIES` constant used anywhere?

Reply:
`SUPPORTED_DATACITE_METADATA_PROPERTIES ` had been removed from the Datacite class.